### PR TITLE
Introduce Node API endpoints to follow REST standards - Closes #862

### LINKED ISSUE #862: Introduce Node API endpoints to follow REST standards
Parent Issue: #225 

In order to meet the requirements of the 1.0.0 API, `/api/node` endpoints should be introduced - link: https://app.swaggerhub.com/apis/LiskHQ/Lisk/1.0.14

### PR DESCRIPTION
**Pull request by MaciejBaj**
Closes issue: #862
Children issues: #869
Parent issue: #225

### CODE REVIEW

File: test/functional/http/get/node.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	var constantsResponse;
+
+	before(function (done) {
+		http.get('/api/node/constants', function (err, res) {
+			constantsResponse = res.body;
+			done();
+		});
+	});
+
+	it('should return a result containing build = ""', function () {
```
Comment (diego-G) [inline test/functional/http/get/node.js]: I think the descriptions here are redundant. We can have just one it block with all the expectations.

File: test/functional/http/get/node.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
+
+	var constantsResponse;
+
+	before(function (done) {
+		http.get('/api/node/constants', function (err, res) {
```
Comment (diego-G) [inline test/functional/http/get/node.js]: I'd prefer to continue using promisified functions from apiHelpers. I've done this all along the rest offunctional tests.

File: test/functional/http/get/node.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
```
Comment (diego-G) [inline test/functional/http/get/node.js]: I'd create a general describe wrapper `GET /api/node` with two childs `/constans` and `/status`.

**Comment by diego-G** [comment]
Remember to add the new `test/functional/http/get/node.js` files to the index. Otherwise, it won't be tested in Jenkins.

File: test/functional/http/get/node.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+		node.expect(constantsResponse).to.have.property('commit').that.is.a('string').and.has.a.lengthOf(40);
+	});
+
+	it('should return a result containing epoch as a string date', function () {
+		node.expect(constantsResponse).to.have.property('epoch').that.is.a('string');
+		node.expect(new Date(constantsResponse.epoch)).not.to.equal('Invalid Date');
+	});
+
+	it('should return a result containing nethash = "198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d"', function () {
+		node.expect(constantsResponse).to.have.property('nethash').equal('198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d');
```
Comment (SargeKhan) [inline test/functional/http/get/node.js]: Will the nethash always be this, or will it depend on the order of running this test?

File: helpers/constants.js [hunk 5]
```diff
@@ -55,7 +55,9 @@ var constants = {
 		secondSignature: 500000000,
 		delegate: 2500000000,
 		multisignature: 500000000,
-		dapp: 2500000000,
+		dappRegistration: 2500000000,
+		dappWithdrawal: 10000000,
```
[the remark this answers is not in the document]
Comment (MaciejBaj) [reply helpers/constants.js]: https://app.swaggerhub.com/apis/LiskHQ/Lisk/1.0.18

File: test/functional/http/get/node.js (hunk 4 above)
Comment (MaciejBaj) [reply test/functional/http/get/node.js]: nethash will remain the same always - devnet

File: test/functional/http/get/node.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var node = require('../../../node.js');
+var getNodeConstantsPromise = require('../../../common/apiHelpers').getNodeConstantsPromise;
+var getNodeStatusPromise = require('../../../common/apiHelpers').getNodeStatusPromise;
+
+describe('GET /api/node', function () {
+
+	describe('constants', function () {
```
Comment (diego-G) [inline test/functional/http/get/node.js]: `/constans`

File: test/functional/http/get/node.js [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+		it('should return a result containing fees.dappDeposit = 10000000', function () {
+			node.expect(constantsResponse).to.have.nested.property('fees.dappDeposit').equal(10000000);
+		});
+
+		it('should return a result containing fees.data = 10000000', function () {
+			node.expect(constantsResponse).to.have.nested.property('fees.data').equal(10000000);
+		});
+	});
+
+	describe('status', function () {
```
Comment (diego-G) [inline test/functional/http/get/node.js]: `/status`

**Pull request merged**